dockerfiles/startall.sh

```diff
@@ -40,11 +41,15 @@ while [ "$#" -gt 0 ]; do
       p) shift
          PERSIST=$1
          ;;
+      x) shift;
+          DEP_UPDATE_ARG=""
+         ;;
       h) echo "Usage: startall.sh [options]"
          echo "    -d <docker-repo>   Docker repository to fetch images from"
          echo "    -l                 Build image label to use"
-         echo "    -u                 Use "upgrade" rather than "install"
```

Comment:
@richardkchapman there seemed to be a quoting issue I went ahead and changed because I encountered errors when attempting to issue -h commands, please let me know if this needs to be reverted.
Also, I chose a very arbitrary option "x" for this directive, please let me know if we can think of a better option. thanks.